tests/projects/SelfContained_Trimming_Test/check.ps1

```diff
@@ -14,7 +14,7 @@ if (-not ($output -eq $expected))
 }
 
 # Checking that FSharp.Core binary is of expected size (needs adjustments if test is updated).
-$expected_len = 251904 # In bytes
+$expected_len = 264704 # In bytes
```

Comment:
This time the increase of trimmed Fsharp.Core is quite big.
Do we know what contributes the most to the code size here?

Reply:
@t-gro we use discriminated unions quite a bunch in FSharp.Core.  This change will have caused their Properties and Nested types to be kept.